jsonnet/openebs-mixin/dashboards/openebs/localpv-workload.json

```diff
@@ -1,4216 +1,4195 @@
+        "options": [],
+        "query": "label_values(namespace_workload_pod:kube_pod_owner:relabel{workload=\"$workload\"},pod)",
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "$datasource",
+        "definition": "label_values(kube_pod_spec_volumes_persistentvolumeclaims_info{pod=~'$pod'},persistentvolumeclaim)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "pvcName",
+        "multi": false,
+        "name": "pvcName",
+        "options": [],
+        "query": "label_values(kube_pod_spec_volumes_persistentvolumeclaims_info{pod=~'$pod'},persistentvolumeclaim)",
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "$datasource",
+        "definition": "label_values(kube_persistentvolumeclaim_info{persistentvolumeclaim='$pvcName'},volumename)",
+        "description": null,
+        "error": null,
+        "hide": 2,
+        "includeAll": true,
+        "label": "pvName",
+        "multi": false,
+        "name": "pvName",
+        "options": [],
+        "query": "label_values(kube_persistentvolumeclaim_info{persistentvolumeclaim='$pvcName'},volumename)",
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "$datasource",
+        "definition": "label_values(openebs_size_of_volume{name='$pvName'},device)",
+        "description": null,
+        "error": null,
+        "hide": 2,
+        "includeAll": true,
+        "label": "device",
+        "multi": false,
+        "name": "device",
+        "options": [],
+        "query": "label_values(openebs_size_of_volume{volumename='$pvName'},device)",
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-12h",
+    "to": "now"
+  },
+  "timepicker": {
+    "refresh_intervals": [
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ],
+    "time_options": ["5m", "15m", "1h", "6h", "12h", "24h", "2d", "7d", "30d"]
+  },
```

Comment:
What are the changes exactly in this commit?

Reply:
cStor dashboards were updated for helm charts which I forgot to update for mixin so updated the cStor dashboards and few formatting changes.